db/models.py

```diff
@@ -1 +1,29 @@
 from django.db import models
+from django.db.models import ForeignKey
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
```

Comment:
Consider adding a `related_name` attribute to the `race` ForeignKey in the `Skill` model for better reverse querying capabilities, as you have done in the migrations.